### PR TITLE
Flexibus/#379/teams page design

### LINKED ISSUE #379: Teams view as tiles
**#380 depends on this for tile component**
**#374 depends on this for tile component and team logic**

New Layout for "Teams" Page:

Header and footer are out of scope.
If the team has a team-picture, then tile has this picture. If not, then we use 1 of 2 default pictures (they can be easily found by link in Flexibus channel)

In wide-screens every tile has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row. (**this is not shown in pictures**)
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

- [ ] create a reusable component for the tiles that can be used in #380.

- whole tile should be clickable
- no priority for teams
- 'Deine Teams' only shows teams that user is a member of
- 'Deine Teams' is subset of 'Alle Teams' (not disjunct)

- 'Deine Teams' block is not shown when there is no data

### PR DESCRIPTION
**Pull request by emanuel-metzenthin**
Integrated requested changes, closes #379.

### CODE REVIEW

File: app/assets/stylesheets/navbar.css [hunk 1]
```diff
@@ -169,6 +169,7 @@ img.avatar {
 
 .container, .container-fluid{
     max-width: 1200px;
+    width: 100% !important;
```
Comment (deeps96) [inline app/assets/stylesheets/navbar.css:172]: Why is this change assigned to this ticket? - I think it belongs to #183.

**Review by deeps96 — CHANGES_REQUESTED** [review]
Remove this selection:
image

There is still some intend, which was explicitly not requested by PO. @zhamanakov Is this ok?

image

Intendation is slightly bigger on square images.

Overall code looks okay to me.

File: app/assets/stylesheets/navbar.css (hunk 1 above)
Comment (emanuel-metzenthin) [reply app/assets/stylesheets/navbar.css:172]: That was part of Dimas requested changes for the event tiles as bootstrap somehow always set the container width to 70%.

Comment (deeps96) [reply app/assets/stylesheets/navbar.css:172]: FYI: The container stuff is generally wanted and comes from Materialize.

**Review by deeps96 — APPROVED** [review]

**Pull request merged**